plugins/home/src/plugin.ts

```diff
@@ -172,3 +187,26 @@ export const HeaderWorldClock = homePlugin.provide(
     },
   }),
 );
+
+/**
+ * Display top visited pages for the homepage
+ * @public
+ */
+export const HomePageTopVisited = homePlugin.provide(
+  createCardExtension<Partial<VisitedByTypeProps>>({
```

Comment:
Question (non-blocking): 
Could some of these properties be converted into widget configurations (I mean homepage card extensions can have a modal for configuration) and the user for example can fill in a form with how many items they want to see?

Reply:
Yes, this was in my list for a while. I would like the user to be able to customize how many items they see on screen for example. I think this would be a great followup or contribution.

Reply:
That will be great 🙂